consensus/bitcoin-ng.md

```diff
@@ -52,6 +52,10 @@ There is major challenge in providing liveness and integrity of the chain in the
      └────────┘                        └────────┘
       Leader 2 Award: Governance(Coinbase) + 0.6*(FeeN1+FeeN2) + 0.4*FeeN3
```

Comment:
Not fully in scope of this PR though this computation shall clarify if floating point or integer math. Tracked as https://www.pivotaltracker.com/story/show/160090121